app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+  belongs_to :user, class_name: 'User', foreign_key: 'users_id'
```

Comment:
Please add an association between an `author` and a `like` so a `like` belongs to an `author`, not a `user`.
`belongs_to :author, class_name: 'User'`